lntest/itest/lnd_test.go

```diff
@@ -10245,16 +10247,6 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("bob's htlc should have advanced to the second stage: %v", err)
```

Comment:
This should not be true anymore? If the htlc timeout tx is not confirmed yet, the HTLC shouldn't be in stage 2.

Reply:
The HTLC timeout transaction has already confirmed by the time we're here.

Reply:
You're right (I think). I guess the comments were just wrong before, saying that the htlc sweep was already confirmed at this point